guides/09_Amazon_Voice_Focus.md

```diff
@@ -283,9 +283,27 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 ## Enabling Voice Focus with Echo Reduction
 
-Enabling Voice Focus with Echo Reduction is a two step process: 
+Enabling Voice Focus with Echo Reduction is a three step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Changing the namespace to Amazon Chime Meetings SDK
+
+To create meetings, the following line of code is used to address the namespace:
+
+```typescript
+const chimeMeetings = AWS.Chime();
+```
+
+To use the Echo Reduction feature, developers need to migrate to the Amazon Chime Meetings SDK by updating this line of code with the new namespace and the endpoint region:
```

Comment:
can we just say region?

Reply:
It is mentioned as endpoint region in the Amazon Chime Meetings SDK